src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -49,29 +51,17 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _) {
         this.eventNames = eventNames;
         this.options = $.extend({}, defaults, options);
         overlaysControl = new CAC.Map.OverlaysControl();
-        map = new cartodb.L.map(this.options.id, { zoomControl: false })
-                           .setView(this.options.center, this.options.zoom);
 
-        tabControl = options.tabControl;
-        homepage = options.homepage;
+        this.itineraryControl = new CAC.Map.ItineraryControl({map: null});
 
-        // put zoom control on top right
-        zoomControl = new cartodb.L.Control.Zoom({ position: 'topright' });
-
-        initializeBasemaps();
-
-        if (!homepage) {
-            // hide zoom control on home page view
-            zoomControl.addTo(map);
-            // delay loading overlays and layer switcher
-            initializeOverlays();
-            initializeLayerControl();
+        // only load map tiles if map visible
+        if (!homepage || $(this.options.selector).is(':visible')) {
```

Comment:
It _may_ be better for this condition to check page width rather than visibility of the map div? I could see the styles changing in the future and this silently breaking, whereas if we check for page width it would always work.